flag.go

```diff
@@ -155,6 +157,7 @@ type Countable interface {
 type VisibleFlag interface {
 	// IsVisible returns true if the flag is not hidden, otherwise false
 	IsVisible() bool
+	IsDefaultVisible() bool
```

Comment:
I think this should be moved into DocGenerationFlag interface since it doesnt affect the visibility of the flag as much as helping in DocGeneration.

Reply:
But isn't the same true for "IsVisible"?

Reply:
Additionally: I have no clue, how to access `HideDefault` from DocGenerationFlag

Reply:
Just move the IsDefaultVisible() bool function to DocGenerationFlag and thats it. No other changes needed on your side.

Reply:
Done.

Thanks.

Reply:
Your pipelines failed because the output now changed.